include/rocksdb/cache.h

```diff
@@ -255,12 +255,19 @@ class Cache {
   // returns the memory size for the entries residing in the cache.
   virtual size_t GetUsage() const = 0;
 
+  // returns the memory size for the high priority entries residing in the
```

Comment:
Thanks @koldat, in general this looks good to me.

However, this PR is a bit more than just exposing things to RocksJava... I note you are also adding new C++ API methods to expose values in the C++ API. I think therefore we should just check with @siying that he is happy for these values to be added to the public API...